roadmap/implementers-guide/src/types/network.md

```diff
@@ -0,0 +1,119 @@
+# Network Types
+
+These types are those that are actually sent over the network to subsystems.
+
+## Universal Types
+
+```rust
+type RequestId = u64;
+type ProtocolVersion = u32;
```

Comment:
FWIW, this is not used anywhere currently

Reply:
That's fine, because we don't have more than one protocol version yet.